docs/guide/result.md

```diff
@@ -25,12 +25,12 @@ $result->isAttributeValid('name');
 ## Errors
 
 Most of the time telling only the status of validation is not enough. There are multiple methods to get detailed errors 
-list with their data from result. The difference between them is in the grouping, filtering and representation of every 
-error. Choose one to fit your needs depending on a situation.
+list with their data from the result. The difference between them is in the grouping, filtering, and representation of every 
```

Comment:
Why add the comma before "and"?

Reply:
https://www.grammarly.com/blog/comma-before-and/

Reply:
https://github.com/yiisoft/yii2/blob/master/docs/documentation_style_guide.md#general-style

Does Oxford comma apply to Yii3 too?

Reply:
Doesn't matter for now but it shouyld be consistend and part of the style guide we'll choose.